### ISSUE TITLE
'image' required in service definition

### ISSUE THREAD

**Issue by ortizman** [ktomk/pipelines#10]
This failure occurs when I have defined 'docker' as a service in order to increase the RAM size.

bitbucket-pipelines.yml

```
....
definitions: 
  services:
    docker:
      memory: 3072

....
```

Error

```
    container......: pipelines-1.Format-Validation.default.pts_objects
pipelines: file parse error: 'image' required in service definition
```

**Comment by ktomk** [comment]
I see. This is an error and an unneeded one. thanks for the report. For the local run, the memory requirements would not be effective, so for local testing, you don't need this (the docker service is always available).

This should be easy to patch with the example at hand.

**Comment by ktomk** [comment]
@ortizman next release should have a fix. a [test-build has temporarily available some artifacts](https://github.com/ktomk/pipelines/actions/runs/753778529) - regardless of which configuration the phar was build, they are all identical, so you can pick any of them, e.g. [phar-build-Linux-7.4](https://github.com/ktomk/pipelines/suites/2507974880/artifacts/54228170) if interested.

**Comment by ktomk** [comment]
Version [0.0.54 has been released](https://github.com/ktomk/pipelines/releases/tag/0.0.54) in the hope to address this issue.

If there is further feedback, please feel free to continue on this issue here, e.g. a regression or some problem with the fix or just confirmation the fix does it for you.

Thanks again for bringing this issue to our attention.
